appIcons.js

```diff
@@ -297,6 +312,13 @@ const MyAppIcon = new Lang.Class({
             this.parent();
         this._onFocusAppChanged();
         this._updateCounterClass();
+        
+        // Enable / Disable the backlight of running apps
+        if (this.app.state !== Shell.AppState.STOPPED && this._dtdSettings.get_boolean('unity-backlit-items') === true) {
```

Comment:
Could we also check for the number of windows? This would make the feature compatible with workspace isolation!

`&& getInterestingWindows(this.app, this._dtdSettings).length > 0`